src/client.ts

```diff
@@ -36,8 +36,8 @@ export type ScubaClientParameters = Omit<
 };
 
 export type ScubaMetrics = {
-    objectsTotal: number;
-    bytesTotal: number;
+    objectsTotal: string;
+    bytesTotal: string;
```

Comment:
should we not use BigInt in the (typescript) api, so that it is easy to consume?
and the client can then handle the (de)serialization to/from string when needed (and support numbers as well, for backwards compatibility in case it is used with an older scuba)